### ISSUE TITLE
[Enhancement] parse the sdk docs/code

### ISSUE THREAD

**Issue by jfmherokiller** [siblount/JzSense-Generator#6]
I saw "Find a way to capture all of the hidden functions not listed in DAZ version 3 and version 4 documentation. " on the main page and I think alot of the hidden functions are in the sdk docs.

**Comment by siblount** [comment]
Good catch! Thanks for that. Although it doesn't capture all of the functions, it have some functions and classes not mentioned in the latest JzSense update. I will be taking a look at it.

**Comment by jfmherokiller** [comment]
Another thing you can try is checking the export symbols of DzCore.dll because thats where I think the QTScript to DAZ binding is located

**Comment by jfmherokiller** [comment]
i mean run something like `dumpbin /EXPORTS` on it.  I have been currently studying the script engine currently to see if i can make some kind of .net bridge for it. 

TLDR I want to write plugins in c# vs writing them in c++,

**Comment by siblount** [comment]
> i mean run something like `dumpbin /EXPORTS` on it. I have been currently studying the script engine currently to see if i can make some kind of .net bridge for it.
> 
> TLDR I want to write plugins in c# vs writing them in c++,

Interesting. I did not know that. Thanks for the information, I'll take a peek at it when I have some free time.
